services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+from typing import Optional
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: Optional[datetime] = None
+                        ) -> MovieSession:
+    queryset = MovieSession.objects.all()
+    if session_date is not None:
+        queryset = queryset.filter(show_time__date=session_date)
+
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int, ) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: Optional[str] = None,
+        movie_id: Optional[int] = None,
+        cinema_hall_id: Optional[int] = None
+) -> None:

```

Comment:
You have different formatting in this piece of code and in other functions. Please, format everything in one way. 
You can try using black formatter to automate this process. 
Also, this will fix `'` in your migrations file.